### PR TITLE
Prevent config to be bundled in client.

### PR DESCRIPTION
**Pull request by langpavel**

### CODE REVIEW

**Comment by langpavel** [comment]
@koistya Can be safely merged to both feature branches

**Comment by koistya** [comment]
Thanks! I'm just thinking, is it the best approach.. another solution might be to restrict this file from being compiled in client-side bundle via webpack config..

**Comment by langpavel** [comment]
@koistya Hmm that is also possible but must be done in way that user of RSK will be correctly errored when trying to do this

**Comment by langpavel** [comment]
@koistya Do you merge this or do you prepare different solution?

**Comment by koistya** [comment]
@langpavel give me some more time to think about it please

**Comment by fchienvuhoang** [comment]
like 💃

**Comment by fchienvuhoang** [comment]
love this project thanks so much @koistya

**Comment by langpavel** [comment]
Ok :-)

**Comment by langpavel** [comment]
@koistya Just picking this up as I see chat on gitter :-)

File: src/config.js [hunk 1]
```diff
@@ -10,6 +10,19 @@
 /* eslint-disable max-len */
 /* jscs:disable maximumLineLength */
 
+/**
+ * WARNING: Do not include this file in client build.
+ */
+const DO_NOT_USE_THIS_FILE_IN_CLIENT = `
+Configuration file contaions sensitive information and
```
Comment (aaron-harvey) [inline src/config.js]: contains

**Comment by langpavel** [comment]
@koistya, @frenzzy Any new opinion on this? I'm voting :+1:

**Comment by ulani** [comment]
@langpavel thank you very much for this PR! Unfortunately, we have close it due to inactivity. Feel free to re-open it or [join](https://discord.com/invite/2nKEnKq) our [Discord](https://discord.com/invite/2nKEnKq) channel for [discussion](https://github.com/kriasoft/react-starter-kit/discussions/1950).

NOTE: The `main` branch has been updated with React Starter Kit v2, using JAM-style architecture.

**Pull request closed without merging**